packages/radix-ui-themes/src/components/checkbox-card-group.css

```
@@ -0,0 +1,158 @@
+
+.rt-CheckboxCardGroupRoot {
+  line-height: var(--line-height);
+  letter-spacing: var(--letter-spacing);
+}
+
+.rt-CheckboxCardGroupItem {
+  padding: var(--radio-card-group-item-padding);
+  border-radius: var(--radio-card-group-item-radius);
+  gap: var(--radio-card-group-item-gap);
+}
+
+@breakpoints {
+  .rt-CheckboxCardGroupRoot {
+    /* 40px height for the card with one line of text */
+    &:where(.rt-r-size-1) {
+      font-size: var(--font-size-2);
+      --line-height: var(--line-height-2);
+      --letter-spacing: var(--letter-spacing-2);
+      --radio-card-group-item-padding: calc(var(--space-3) / 1.2) var(--space-3);
+      --radio-card-group-item-radius: var(--radius-3);
+      --radio-card-group-item-gap: var(--space-2);
+    }
+    /* 48px height for the card with one line of text */
+    &:where(.rt-r-size-2) {
+      font-size: var(--font-size-2);
+      --line-height: var(--line-height-2);
+      --letter-spacing: var(--letter-spacing-2);
+      --radio-card-group-item-padding: calc(var(--space-4) * 0.875) var(--space-4);
+      --radio-card-group-item-radius: var(--radius-3);
+      --radio-card-group-item-gap: var(--space-3);
+    }
+    /* 64px height for the card with one line of text */
+    &:where(.rt-r-size-3) {
+      font-size: var(--font-size-3);
+      --line-height: var(--line-height-3);
+      --letter-spacing: var(--letter-spacing-3);
+      --radio-card-group-item-padding: calc(var(--space-5) / 1.2) var(--space-5);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-4);
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * VARIANTS                                                                                        *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-CheckboxCardGroupItem {
+  position: relative;
+  /* pseudo element for focus outline */
+  &::before {
+    content: '';
+    display: block;
+    position: absolute;
+    inset: 0;
+    border-radius: inherit;
+    pointer-events: none;
+  }
+}
+
+/* surface */
+
+.rt-CheckboxCardGroupRoot:where(.rt-variant-surface) {
+  :where(.rt-CheckboxCardGroupItem) {
+    background-color: var(--color-surface);
+    box-shadow: inset 0 0 0 1px var(--gray-a5);
+
+    @media (hover: hover) {
+      &:where(:not(:has(:disabled)):hover) {
+        box-shadow: inset 0 0 0 1px var(--gray-a7);
+      }
+    }
+
+    &:where(:has(:disabled)) > * {
+      color: var(--gray-a9);
+    }
+
+    &:has(:focus-visible)::before {
+      outline: 2px solid var(--color-focus-root);
+      outline-offset: 2px;
```

Don't think we should make it look like the Radio Card when it can look like Card

```suggestion
      outline-offset: -1px;
```

---

Ah yeah good point, that treatment is only really needed because of the checked state on radio cards.